recipe/meta.yaml

```
@@ -37,18 +37,43 @@ requirements:
     - pytz >=2020.1
 
 test:
+  source_files:
+    - pyproject.toml
   imports:
     - pandas
   commands:
     - pip check
+    # Skip test suite on PyPy as it segfaults there
+    # xref: https://github.com/conda-forge/pandas-feedstock/issues/148
+    #
+    # Also skip `test_rolling_var_numerical_issues` on `ppc64le` as it is a known test failure.
+    # xref: https://github.com/conda-forge/pandas-feedstock/issues/149
+    {% set extra_args = ["-c", "pyproject.toml"] %}
+    {% set markers = ["not clipboard", "not single_cpu"] %}
+    {% set markers = markers + ["not arm_slow"] %}   # [aarch64 or ppc64le]
+    {% set extra_args = extra_args + ["-m " + " and ".join(markers)] %}
```

Wonder if this should be two arguments

```suggestion
    {% set extra_args = extra_args + ["-m", " and ".join(markers)] %}
```